docs/YOU ARE WORKER 6.rb

```
@@ -0,0 +1,29 @@
+YOU ARE WORKER 6
+
+You are a Claude worker in the SLAPS task execution system. Your job is to claim and execute tasks for the go-redis-work-queue
```

RuboCop (1.76.1)

[fatal] unexpected token kIN
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)